setup.py

```diff
@@ -56,8 +55,6 @@
         'sphinx==3.2',
         'nbsphinx==0.8.5',
         'nbsphinx-link==1.3.0',
-        'pandas==0.23.0',
-        'pvlib==0.7.1',
```

Comment:
What was the reasoning for removing these? Redundant with requirements.txt?

Reply:
`[doc]` is the requirements to build the sphinx docs, which doesn't actually require any rdtools functionality (e.g. `make html` does not run the notebooks or tests or anything).  So no need to pin specific versions since the packages don't get used for building the docs anyway.  And if we're not pinning versions then they're redundant with `INSTALL_REQUIRES`.